crypto/conf/conf_mod.c

```diff
@@ -522,7 +522,7 @@ int CONF_parse_list(const char *list_, int sep, int nospc,
     lstart = list_;
     for (;;) {
         if (nospc) {
-            while (*lstart && isspace((unsigned char)*lstart))
+            while (*lstart && ossl_isspace((unsigned char)*lstart))
```

Comment:
Does this mean config files are expected to be ASCII formatted, rather than localized?